src/pages/umi/getting-started/index.md

```diff
@@ -53,10 +56,172 @@ export async function myFunction(
 }
 ```
 
-## For testing
+{% /totem-accordion %}
+
+{% /totem %}
+
+### For testing
 
 Also note that Umi comes with a testing bundle that can help both end-users and library authors to test their code. For instance, it includes a `MockStorage` implementation used for both the `UploaderInterface` and the `DownloaderInterface` so you can reliably test your code without having to rely on a real storage provider.
 
-```sh
-npm install @metaplex-foundation/umi @metaplex-foundation/umi-bundle-tests
-```
\ No newline at end of file
+```
+npm i @metaplex-foundation/umi
+```
+
+```
+npm i @metaplex-foundation/umi-bundle-tests
+```
+
+## Umi Basics
+
+In this section, we'll cover the essential steps to get started with Umi:
+- [Creating Umi and Connecting to an RPC](/umi/getting-started#connecting-to-an-rpc)
+- [Connecting a Wallet](/umi/getting-started#connecting-a-wallet)
+- [Registering Programs and Clients](/umi/getting-started#registering-programs-and-clients)
+
+### Connecting to an RPC
+
+Solana has different clusters (e.g., Mainnet-beta, Devnet, Testnet, ...) that serve various purposes, each with dedicated API nodes to handle RPC requests.
+
+Connecting Umi to a cluster of choice is is as simple as creating an umi instance since the RPC endpoint is passed as the first argument. If you're connecting to `Mainnet`, it's recommended to use a dedicated RPC endpoint from a Solana RPC provider instead of the public endpoint (https://api.mainnet-beta.solana.com) due to its limitations.
+
+To create a Umi instance, import the `createUmi` function and provide your RPC endpoint. Optionally, you can also specify the commitment level as the second argument.
+
+```ts
+import { createUmi } from '@metaplex-foundation/umi-bundle-defaults'
+
+const umi = createUmi('<RPC-Endpoint>', '<Commitment-Level>')
+```
+
+Here are all the public endpoints for the different clusters:
+
+{% totem %}
+
+{% totem-accordion title="Mainnet-beta" %}
+
+```ts
+const umi = createUmi('https://api.mainnet-beta.solana.com')
+```
+
+{% /totem-accordion %}
+
+{% totem-accordion title="Devnet" %}
+
+```ts
+const umi = createUmi('https://api.devnet.solana.com')
+```
+
+{% /totem-accordion %}
+
+{% totem-accordion title="Testnet" %}
+
+```ts
+const umi = createUmi('https://api.testnet.solana.com')
+```
+
+{% /totem-accordion %}
+
+{% totem-accordion title="Localnet" %}
+
+```ts
+const umi = createUmi('http://127.0.0.1:8899')
+```
+
+{% /totem-accordion %}
+
+{% /totem %}
+
+### Connecting a Wallet
```

Comment:
The drop down accordian here has 3 titles

```
a new wallet
an existing wallet
a wallet adapter
```

this should be clearer. Existing wallets and walletadpater is essentially the same thing, both are existing wallets.

should be made clear these are file sytem wallets in some way to avoid potentially confusion.